pkg/driver/node_linux.go

```diff
@@ -116,6 +125,31 @@ func (d *nodeService) findDevicePath(devicePath, volumeID, partition string) (st
 	return canonicalDevicePath, nil
 }
 
+func verifyVolumeSerialMatch(canonicalDevicePath, strippedVolumeName string) error {
```

Comment:
I know that we said adding unit tests for the file is too difficult, but you cleanly extracted this check's logic into what looks to be a testable function (I see [this article explains how we would mock that lsblk exec](https://stackoverflow.com/a/71166114)). 

Could we consider adding unit tests for `verifyVolumeSerialMatch`? 

Might be overkill, but would help us get closer to node service unit tests if/when we do refactor `node_linux.go`.